bench/wtperf/wtperf_opt.i

```diff
@@ -203,9 +204,9 @@ DEF_OPT_AS_STRING(table_name, "test", "table name")
 DEF_OPT_AS_BOOL(
   truncate_single_ops, 0, "Implement truncate via cursor remove instead of session API")
 DEF_OPT_AS_UINT32(
-  value_sz_max, 1000, "maximum value size when delta updates are present. Default disabled")
+  value_sz_max, 1000, "maximum value size when delta updates/modify are present. Default disabled")
 DEF_OPT_AS_UINT32(
-  value_sz_min, 1, "minimum value size when delta updates are present. Default disabled")
+  value_sz_min, 1, "minimum value size when delta updates/modify are present. Default disabled")
```

Comment:
The wording for both of these is a bit awkward, especially now with the addition. I suggest:
`... when delta update/modify operations are present.`

Reply:
Fixed.